R/cedta.R

```diff
@@ -22,7 +22,12 @@ cedta.pkgEvalsUserCode = c("gWidgetsWWW","statET","FastRWeb","slidify","rmarkdow
 # cedta = Calling Environment Data.Table-Aware
 cedta = function(n=2L) {
   # Calling Environment Data Table Aware
-  ns = topenv(parent.frame(n))
+  env = parent.frame(n)
+  if (isTRUE(ns$.datatable.aware)) {
```

Comment:
@hadley `ns` doesn't exist here, is this supposed to be `env$.datatable.aware`?

Reply:
Now I don't remember what I was thinking 😞 But yes using `env$.datatable.aware` seems reasonable.